pyscriptjs/src/remote_interpreter.ts

```diff
@@ -45,6 +47,17 @@ type PyScriptInternalModule = ProxyMarked & {
     schedule_deferred_tasks(): void;
 };
 
+// TODO: find a way to provide this definition more broadly
+type PyodideHelper = {
+    type: string;
+    runtime: InterpreterInterface;
```

Comment:
I have been away for too long, but a couple of months a go we decided to rename `runtime` as `interpreter`, will we go back to runtime?

To be honest I don't really mind whatever we choose, just wondering 😄

Reply:
https://github.com/pyscript/pyscript/issues/1533